axelar-chains-config/info/mainnet.json

```diff
@@ -368,7 +368,7 @@
       },
       "gasOptions": {
         "gasLimit": 8000000,
-        "gasPriceAdjustment": 1.4
+        "gasPriceAdjustment": 1.8
```

Comment:
Do we need to adjust Fantom in this PR?

Reply:
ah, needed it for something else, will update it in another PR